packages/abi-coder/test/Interface.test.ts

```diff
@@ -131,6 +131,21 @@ describe('Abi interface', () => {
     });
   });
 
+  describe('messagesTypes', () => {
+    it('sets messagesTypes properties just fine', () => {
```

Comment:
IMO this test isn't necessary as it tests the generated abi's properties. We don't have to test the abi, we just follow the spec. It's the forc repository's job to ensure that message types are there and they'd have some equivalent of this test somewhere.

Reply:
@nedsalk If you feel strongly that it doesn't make sense, I'm happy to remove it. I added it because, at the moment, we don't have anything in place to validate the ABI `messageTypes`.

Reply:
I'd say remove them as they don't test anything that we own. We'll add relevant tests while implementing https://github.com/FuelLabs/fuels-ts/pull/3016#discussion_r1729245278.

Reply:
https://github.com/FuelLabs/fuels-ts/pull/3016/commits/bb3a376269326f94ecccc6c1a609463d145d6662